File: Benchmarking_Languages_Project/Scala_Tests/Test8.scala
var timing = DenseVector.zeros[Long](15)
for(i <- 0 until 15){
	
	val start = System.currentTimeMillis()
	val T8 = DenseMatrix.rand(2500,2500)
	val T8_1 = det(T8)
	val finish = System.currentTimeMillis()
	timing(i) = (finish - start)
}

println(mean(timing)/1000)

File: Benchmarking_Languages_Project/Python_Tests/Test8.py
from scipy import *
from numpy import *
from time import *


timing = zeros(15)
for i in range(15):
	time1 = time()
	T8 = matrix(random.normal(size=(2500,2500)))
	T8_1 = linalg.det(T8)
	timing[i]=time()-time1
print mean(timing)

